src/Components/InputSearchField/InputSearchField.tsx

```diff
@@ -0,0 +1,59 @@
+// import { Input } from '@mui/material';
+import React from 'react';
+
+// const InputSearchField = () => {
+//   return (
+//     <Input aria-label="Demo input" placeholder="Type something…" />
+//   );
+// }
+
+// export default InputSearchField;
+
+import { Input as BaseInput } from '@mui/base/Input';
+import { styled } from '@mui/system';
+import { blue, grey } from '../../CustomTheme/Colors/colors.tsx';
+
+const Input = React.forwardRef(function CustomInput(
```

Comment:
Nothing wrong with this but you can directly use MUI Input Components like Autocomplete or Textfield to create your custom component, as is the practise